### PR TITLE
head: add Russian translation

### PR DESCRIPTION
**Pull request by dremdem**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ru/common/head.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# head
+
+> Output the first part of files.
```
Comment (mfrw) [inline pages.ru/common/head.md:3]: Wondering why this line is not in Russian ?

Comment (dremdem) [reply pages.ru/common/head.md]: Nice catch! Fixed.

File: pages.ru/common/head.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# head
+
+> Выводит первую часть файлов.
+> Больше информации: <https://www.gnu.org/software/coreutils/head>.
+
+- Выводит первые несколько строк из файла:
+
+`head -n {{count_of_lines}} {{filename}}`
```
Comment (navarroaxel) [inline pages.ru/common/head.md:8]: Please translate the placeholders for get this easier for non-English speakers.

Comment (dremdem) [reply pages.ru/common/head.md:8]: Sure, one moment.

Comment (dremdem) [reply pages.ru/common/head.md]: Done.

**Review by navarroaxel — APPROVED** [review]

File: pages.ru/common/head.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# head
+
+> Выводит первую часть файлов.
+> Больше информации: <https://www.gnu.org/software/coreutils/head>.
+
+- Выводит первые несколько строк из файла:
```
Comment (EmilySeville7cfg) [inline pages.ru/common/head.md:6]:
```suggestion
- Вывести первые несколько строк из файла:
```

Everything is okay, just one suggestion. Also change other descriptions as this one. ;)

Comment (dremdem) [reply pages.ru/common/head.md]: Changed

**Review by EmilySeville7cfg — APPROVED** [review]

**Pull request merged**